packages/client-twitter/src/interactions.ts

```diff
@@ -282,6 +290,37 @@ export class TwitterInteractionClient {
             this.client.saveRequestMessage(message, state);
         }
 
+        const pizzaCheck = `
```

Comment:
Also, should this directly be added to the twitter client and if so should it be configurable as it seems like overkill to always run